### PR TITLE
build: add update-versions and release script

### PR DESCRIPTION
**Pull request by harbu**
## Summary

Add two bash scripts to make releasing packages just a tad bit easier.

Script `update-versions.sh` automatically edits the package.json versions appropriately using `jq`. 

Usage:
```bash
./update-versions.sh 7.1.3
```

Script `release.sh` releases all sub-packages (except broker). Currently all publish commands are with `--dry-run` as I have not used this script to publish for real. Remove those flags once the script has been tested in practice. (I have run it with `--dry-run` ofc.)

Usage:
```bash
./release.sh latest
```

## Limitations and future improvements

- My bash scripting skills are not great so feedback is very welcome indeed :smile: 

## Checklist before requesting a review

- [x] Is this a breaking change? If it is, be clear in summary.
- [x] Read through code myself one more time.
- [x] Make sure any and all `TODO` comments left behind are meant to be left in.
- [ ] Has reasonable passing test coverage?
- [ ] Updated changelog if applicable.
- [ ] Updated documentation if applicable.

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
Comment (teogeb) [inline release.sh:1]: Could maybe add a prompt which ask whether releasing beta or not? (Or automatically analyze that from the current version number)

File: update-versions.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+for pkg in $PKG_DIRS
+do
+    tmp=$(mktemp)
+    if [[ "$pkg" != "broker" ]]; then
+        jq ".version=\"$1\"" packages/$pkg/package.json > $tmp
+    else
+        cat packages/$pkg/package.json > $tmp
+    fi
+    for dep in $PKG_NAMES
+    do
```
Comment (teogeb) [inline update-versions.sh:43]: Maybe a bit simpler/readable version:

```
#!/bin/bash

VERSION=$1
FILE_NAME=package.json 

if [[ "$1" == "" ]]; then
    echo 'usage: update-versions.sh <SEMVER>'
    exit 1
fi

PKG_DIRS="utils
test-utils
protocol
proto-rpc
dht
network
network-tracker
client
cli-tools
broker"

PKG_NAMES="@streamr/utils
@streamr/test-utils
@streamr/protocol
@streamr/proto-rpc
@streamr/dht
@streamr/network-node
@streamr/network-tracker
streamr-client
@streamr/cli-tools
"

# Update package versions
for pkg in $PKG_DIRS
do
    FILE_NAME=packages/$pkg/package.json
    TMP_FILE=$(mktemp)
    if [[ "$pkg" != "broker" ]]; then
        jq --arg versionNumber $VERSION '.version |= $versionNumber' $FILE_NAME > $TMP_FILE
        mv $TMP_FILE $FILE_NAME
    fi
    for PKG_NAME in $PKG_NAMES
    do
        jq --arg versionNumber $VERSION --arg PKG_NAME $PKG_NAME '
        if .dependencies[$PKG_NAME]? then .dependencies[$PKG_NAME] |= $versionNumber
        elif .devDependencies[$PKG_NAME]? then .devDependencies[$PKG_NAME] |= $versionNumber
        else . end' $FILE_NAME > $TMP_FILE
        mv $TMP_FILE $FILE_NAME
    done
done
```

File: release.sh (hunk 1 above)
Comment (harbu) [reply release.sh:1]: Added

File: update-versions.sh (hunk 2 above)
Comment (harbu) [reply update-versions.sh:43]: Merged

**Pull request merged**